azure-pipelines.yml

```diff
@@ -34,4 +34,9 @@ jobs:
 - template: build-pipeline.yml
   parameters:
     # There's a test inside this repo that requires Qt.py
-    extra_test_dependencies: Qt.py
+    extra_test_dependencies:
+    - Qt.py
```

Comment:
I grokked how to use mappings and arrays with templates, so we're better off using arrays and mappings as it allows us greater flexibility when parsing that data in templates. This is the only repo where we've used this extra_test_dependencies feature so far, so we're not going to break anyone.